tests/test_lab_evaluate.py

```diff
@@ -0,0 +1,466 @@
+                "models/instructlab/granite-7b-lab",
+                "--base-model",
+                "models/instructlab/granite-7b-lab",
+                "--sdg-path",
+                "generated",
+            ],
+        )
+        assert run_mock.call_count == 2
+        assert result.exit_code == 0
+        expected = textwrap.dedent(
+            """\
+            # KNOWLEDGE EVALUATION REPORT
+
+            ## BASE MODEL
+            models/instructlab/granite-7b-lab
+
+            ## MODEL
+            models/instructlab/granite-7b-lab
+
+            ### AVERAGE:
+            -0.1 (across 5)
+
+            ### IMPROVEMENTS:
+            1. task1 (+0.1)
+            2. task3 (+0.1)
+
+            ### REGRESSIONS:
+            1. task2 (-0.1)
+            2. task4 (-0.1)
+
+            ### NO CHANGE:
+            1. task5
+            """
+        )
+        assert result.output == expected
+
+
+def test_no_model_mt_bench():
+    runner = CliRunner()
+    with runner.isolated_filesystem():
+        result = runner.invoke(
+            lab.ilab,
+            [
+                "--config=DEFAULT",
+                "model",
+                "evaluate",
+                "--benchmark",
+                "mt_bench",
+            ],
+        )
+        assert (
+            result.output
+            == "Benchmark mt_bench requires the following args to be set: ['model', 'judge-model']\n"
+        )
+        assert result.exit_code != 0
+
+
+def test_missing_benchmark():
+    runner = CliRunner()
+    with runner.isolated_filesystem():
+        result = runner.invoke(
+            lab.ilab,
+            [
+                "--config=DEFAULT",
+                "model",
+                "evaluate",
+            ],
+        )
+        assert "Missing option '--benchmark'" in result.output
+        assert result.exit_code != 0
+
+
+def test_invalid_model_mt_bench():
+    runner = CliRunner()
+    with runner.isolated_filesystem():
+        result = runner.invoke(
+            lab.ilab,
+            [
+                "--config=DEFAULT",
+                "model",
+                "evaluate",
+                "--benchmark",
+                "mt_bench",
+                "--model",
+                "invalid",
+                "--judge-model",
+                "models/instructlab/merlinite-7b-lab",
+            ],
+        )
+        assert "Failed to determine backend:" in result.output
+        assert result.exit_code != 0
+
+
+def test_invalid_model_mmlu():
+    runner = CliRunner()
+    with runner.isolated_filesystem():
+        result = runner.invoke(
+            lab.ilab,
+            [
+                "--config=DEFAULT",
+                "model",
+                "evaluate",
+                "--benchmark",
+                "mmlu",
+                "--model",
+                "invalid",
+            ],
+        )
+        # TODO: This error could use some work
+        assert "is not a valid model" in str(result.exception)
```

Comment:
Should you make this similiar to the `test_invalid_model_mt_bench` error?: `assert "Failed to determine backend:" in result.output`